packages/core/contracts/common/implementation/AncillaryData.sol

```diff
@@ -98,7 +98,7 @@ library AncillaryData {
 
     /**
      * @notice Helper method that returns the left hand side of a "key:value" pair plus the colon ":" and a leading
-     * comma "," if the `currentAncillaryData` is not empty. The return value is intended to be appended as a prefix to
+     * comma "," if the `currentAncillaryData` is not empty. The return value is intended to be prepended as a prefix to
```

Comment:
### Pull Request [3188](https://github.com/UMAprotocol/protocol/pull/3188/)

This PR addresses the comments from [our review of PR 3061](#3061) and [our batch review of PRs 3054, 3082, and 3092](#3054). We have the following comment:

- The [comment](https://github.com/UMAprotocol/protocol/blob/dd211c4e3825fe007d1161025a34e9901b26031a/packages/core/contracts/common/implementation/AncillaryData.sol#L101) explaining the `constructPrefix` function says "appended as a prefix" instead of "prepended".